pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiAuth0LogEvent.java

```diff
@@ -39,6 +39,9 @@ long insertAuth0LogEvent(
             @Bind("data") String data
     );
 
+    @SqlQuery("select log_id from auth0_log_event where tenant = :tenant and log_id = :logId")
+    String findAuth0LogEventByTenantAndLogId(@Bind("tenant") String tenant, @Bind("logId") String logId);
```

Comment:
The name `findAuth0LogEvent...` make me think that it will return a whole event object, but it's just return the `log_id` string. Might be less confusing to name it `findAuth0LogId...` instead.

Reply:
Yes, sure. Fixed